e2e_deployment_files/nested_long_haul_amqp_deployment.template.json

```diff
@@ -2,7 +2,7 @@
   "modulesContent": {
     "$edgeAgent": {
       "properties.desired": {
-        "schemaVersion": "1.0",
+        "schemaVersion": "1.1",
```

Comment:
Why not put both agent and hub at 1.2?

Reply:
For edgeHub, 1.2 is not publicly available and is just used in development, but we need it for broker. To be honest, I don't know what edgeAgent schema 1.2 is or what it means (or if it even exists), but 1.1 is documented, which is why I chose this. But I should definitely ask someone who knows the status of edgeAgent schema versions because if 1.2 does exist we should be testing with it

Reply:
Yeah, the reason why I ask is due to how I think our build pipelines work for creating images. As I think it creates edgehub and edgeagent. But I think Hugues/Bear might know best :)